components/resume.tsx

```
@@ -0,0 +1,84 @@
+import { buttonVariants } from "./ui/button";
+
+import { Briefcase, CalendarPlus2 } from "lucide-react";
+import Image from "next/image";
+import Link from "next/link";
+
+import { env } from "@/app/env";
+import { type Role } from "@/lib/constants";
+
+const Role = ({ role }: { role: Role }) => {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Role'. Consider to delete it or rename it.

'Role' is defined here:


(lint/suspicious/noRedeclare)